### DISCUSSION TITLE
webMAN MOD 1.47.46

### DISCUSSION THREAD

**Discussion by aldostools** [aldostools/webMAN-MOD#1040] [General]
⭐ Install on 4.75-4.91 (CFW/HFW/CEX/DEX/PEX/D-PEX/DECR)
🧹 Added clear webMAN cache files to XMB menu 🆕
🔉 Added sound notification after copy >1GB or after scan games (uncheck BEEP in /setup.ps3 if you want to disable it) 🆕
☑️ PS2 config database updates with fixes by @mrjaredbeta, @agrippa, @kozarovv.
🚀 Cumulative update (includes over 82 commits to the suite of tools after initial release of 1.47.45).

<hr>
💡 <b>If you like webMAN MOD, then feel free to contribute with a *small donation</font>*, I'd really appreciate it!<BR>
&nbsp;  &nbsp; &nbsp; <a href="https://www.paypal.me/aldostools">paypal.me/aldostools</a> or <a href="https://donate.aldostools.org">donate.aldostools.org</a> (Funds are mainly to cover the high server costs)</b><br>
<br>
<a href="http://ps3.aldostools.org/links.html"><b>Download:</b></a> <a href="https://github.com/aldostools/Resources/releases/download/1.0/EP0001-BLES80608_00-COVERS0000000000.pkg">Covers Pack [PS3]</a> &bull; <a href="https://github.com/aldostools/Resources/releases/download/1.0/EP0001-BLES80608_00-COVERS00000RETRO.pkg">Covers Pack [PSX/PS2]</a> &bull; <a href="https://github.com/aldostools/Resources/releases/download/RetroArch_CE/RetroArch_CE.pkg">Retro Arch CE</a> &bull; <a href="https://github.com/aldostools/Resources/releases/tag/Launchers">Launchers</a> &bull; <a href="https://github.com/aldostools/Resources/releases/tag/Addons">Addons</a> &bull; <a href="https://github.com/aldostools/Resources/releases/tag/Themes">Themes</a>  &bull; <a href="https://github.com/aldostools/webMAN-MOD/wiki">Wiki</a> &bull; <a href="https://github.com/aldostools/resources">More</a>

<hr /><em>This discussion was created from the release <a href='https://github.com/aldostools/webMAN-MOD/releases/tag/1.47.46'>webMAN MOD 1.47.46</a>.</em>

**Comment by Tna9rish** [comment]
I wanna ask you something. I'm trying to boot psn games that are located in GAMEI folder from the net when I select it with webman it shows game loaded but no icon in XMB also I already activated thet app_home\ps3_game still nothing shows there and when I click on the app_home it gives me error 80028F14.I'm under HEN and latest webman version which is 1.47.46 . Any help would be great
![bandicam 2024-03-22 01-03-31-113](https://github.com/aldostools/webMAN-MOD/assets/130870965/25acad78-fdff-4028-b3a0-4a572e8c74cb)
and here is a video showing my setup:

https://github.com/aldostools/webMAN-MOD/assets/130870965/f124a0da-8c4c-4447-8daa-93da501c9299

**Comment by aldostools** [comment]
It looks like HEN has issues remapping /dev_bdvd to /app_home. I have to investigate.

The last time I implemented the feature, it was working fine on CFW.

**Comment by Tna9rish** [comment]
> It looks like HEN has issues remapping /dev_bdvd to /app_home. I have to investigate.
> 
> The last time I implemented the feature, it was working fine on CFW.

I tried the russian hen it worked before the official release. But I wanna say thank u for the efforts u did to the ps3-scene ^_^

**Comment by MrStuid** [comment]
Still has issues with freezing on hen.

**Comment by MichaelX17** [comment]
Pal, thanks for your job, you really help us a lot, but this time i have a little issue. I installed the latest version of webMan (1.47.46b) and when i run an ISO game, PKG game or any program my PS3 just turn off, I uninstalled the webMan and all my programs, games and stuff worked fine. I installed the webMan again and everything started turning my PS3 off again, so i had to uninstall the webMan to at least use my PS3. I don't know what is happening, 'cause everything worked absolutely fine until i updated the webMan, even i tried to install the 1.47.46 (That had no problem before) and everything started crashing with that version too... So, i don't know if there is some bug with this version or if is my PS3 that have some issue. I really hope you can help me, man

**Reply by aldostools** [reply]
Check if an older version also cause the issue. If previous versions that were working still cause the crash, then you should try rebuilding the database / file system in recovery menu.

Also check if the issues occurs with the USB devices unplugged.

**Reply by MichaelX17** [reply]
Hi man, i tried those options, but anything worked. I had to backup my files/games/saves, format the disk and install all as new. After that, everything worked fine, but i don't know what happened before, first time this happens to me. Thanks for your help and all your work, keep it up

**Reply by aldostools** [reply]
If you have this folder /dev_hdd0/tmp/system_plugin delete it, reboot and check if that fixes your issues.

If it isn't fixed, backup your saves, licenses and other important files stored in PS3, format internal HDD and reisntall the latest CFW or HFW+HEN.

If you still have the issues after format and reinstall the firmware, then your PS3 could be having a hardware issue.

**Comment by M4uR1CI04lbIT3r** [comment]
How do I install webman mod, I already have hen

**Reply by aldostools** [reply]
In the PC:
1-Go to https://github.com/aldostools/webMAN-MOD/releases
2-Download `webMAN_MOD_1.47.46_Installer.pkg` and prepISO_1.33.pkg
3-Copy the pkg file in the root of a pendrive or USB drive formatted in FAT32 MBR

On the PS3:
4-Turn on the PS3 and Enable HEN.
5-Plug the pendrive or USB drive in any USB port
6-Go to Game column and navigate to Package Manager
7-Select Install Packages > webMAN_MOD_1.47.46_Installer.pkg > press ` ✖ ` to install the installer. Repeat for prepISO_1.33.pkg
8-Find the icon of `webMAN MOD 1.47.46 Installer` and press ` ✖ ` to install webMAN MOD
9-After reboot, enable HEN again. A popup of webMAN MOD will appear and webMAN MOD will start to scan the games.
10-You should see a `webMAN Games` folder. At this point you can delete `webMAN MOD 1.47.46 Installer` since it's not longer used.
11-Go to webMAN Games > webMAN Setup > PS3 webMAN [Setup]. 
12-In the PS3 Internet browser put a check mark on the option `Disable scan for content at startup` and press `Save` button. 
13-Reboot. Enable HEN again and start playing your games.

**Comment by JuanzLzxt** [comment]
Amigo porque estas actualizando el webman mod cada rato tiene problemas ? Porque el mio es el 1.47.46e dime porfavor :(

**Reply by aldostools** [reply]
Porque desde 2014 el webMAN MOD lo actualizo constantemente para hacer mejoras o correcciones... a veces hasta 10 veces en un día.

No estás obligado a actualizar a la ultima versión. De hecho la documentación del wiki lo explica y sugiere que no actualices si todo te funciona correctamente.

![image](https://github.com/aldostools/webMAN-MOD/assets/974897/ad7387cf-33df-4b48-b802-9b23e652fbe8)

**Comment by jlgametube** [comment]
Mano um dia será possível jogar jogos de PS2 no ps3 no modo multiplayer online ?

**Reply by aldostools** [reply]
The PS2 online multi-player is only possible for PS3 phat models with B/C or semi-B/C.
There is literally a physical PS2 embedded in these PS3 models.

ps2_netemu is used by later PS3 models. It uses software emulation.
AFAIK it doesn't have support for network or USB devices in PS2 mode.

**Comment by Zeroprograming** [comment]
Yo por mi parte tengo un erro que hace que cuando ejecuto un juego me apaga la consola, tanto con la version 1.47.46 como la 1.47.45, si no tengo instalado webman me deja ejecutar sin problemas mi juegos pkg, pero una ves lo instalo, ni los .iso ni los .pkg funcionan, ayuda P__P

**Reply by aldostools** [reply]
Los juegos en formato PKG ejecutados de manera directa no tienen relación con webMAN MOD excepto por el fan control.

Si usas PS3HEN debes ir al /setup.ps3 y desactivar el fan control o ponerlo en modo manual o SYSCON. El modo dinamico causa problemas al lanzar juegos.

https://github.com/aldostools/webMAN-MOD/wiki/Setup-Options

**Reply by Zeroprograming** [reply]
Nada, hice lo que me recomendaste, aun asi el error permanece y una ves instalado webMan y estar activo, si trato de ejecutar un Juego en formato PKG o ISO, la consola se apaga y el led deja de dar luz, a pesar de seguir conectada a la red electrica, este error persiste hasta deshabilitar o eliminar webMan

**Reply by aldostools** [reply]
Puedes intentar usando la version Lite para ver si el algun tema de memoria.
Si no funciona, desconozco que puede ser. A mi y a una gran mayoria de usuarios nos funciona correctamente.

**Reply by Zeroprograming** [reply]
Aun con la version Lite, persiste el error, por el momento no queda de otra que espera a que en un futura version de webman logren repararlo, este error solo esta presente en PS3 HFW 4.91 y con la version 1.47.46 y 1.47.45, PS3 SLIM CECH-3001A

**Reply by 7owa5** [reply]
@Zeroprograming pudiste solucionar el problema amigo a mi me pasa exactamente lo mismo me podrias ayudar?

**Reply by ghost** [reply]
> Mesmo com a versão Lite, persista o erro, por enquanto não há queda de outro que espera que em uma futura versão de webman logren o repare, este erro só está presente no PS3 HFW 4.91 e com a versão 1.47.46 e 1.47.45 , PS3 SLIM CECH-3001A

I have the same console model as you and the same thing happens to me upon request, by any chance yours also keeps the Power button LED flashing between red and yellow when trying to mount an ISO game or folder or even when trying to reload the XML?

**Reply by aldostools** [reply]
The flashing leds is performed intentionally in the source code. It's not a bug.

**Reply by ghost** [reply]
Is there a way to remove this?

**Reply by ghost** [reply]
> O piscar de leds é feito intencionalmente no código fonte. Não é um bug.

When I deactivate the option to read the pendrives on the USB, I only let the internal HDD scan and the LEDs don't happen to blink

**Reply by aldostools** [reply]
Yes, compile your own version commenting `led(YELLOW, BLINK_FAST);` everywhere in source code.

The flashing lights are that way by design and it has been working like that for +10 years.

**Reply by ghost** [reply]
> De minha parte, tenho um bug que faz com que quando executo um jogo o console desligue, tanto com a versão 1.47.46 quanto com a 1.47.45, se eu não tiver o webman instalado, ele me permite rodar meus jogos de pacote sem problemas, mas uma vez que eu instalo, nem o .iso nem o .pkg funcionam, ajuda P__P

There was a time when I went to leave a game and the video game went blackout, sometimes when I try to start a game this also happens, another time it happens that the game just stays on a black screen when starting, then the entire video game freezes and I have to force a shutdown, Do you use SSD or HDD?

**Reply by ghost** [reply]
> Sim, compile sua própria versão comentando `led(YELLOW, BLINK_FAST);`em todos os lugares do código-fonte.
> 
> As luzes piscantes são projetadas dessa forma e funcionam assim há mais de 10 anos.

I don't know how to do this either, I don't have a PC, just a cell phone.

**Reply by aldostools** [reply]
Sorry but you will have to ask someone that do it for you. I cannot make changes to the code for particular needs.

**Comment by aldostools** [comment]
Intenta instalando la version 1.47.44 o anterior. Deben funcionar en 4.91

**Comment by pilgisayar63** [comment]
My ps3 firmware version is 4.90 can i install webman 4.91?

**Reply by aldostools** [reply]
Yes. webMAN MOD works on any PS3 with HFW/CFW from 4.75 to 4.91 with Cobra / Mamba / PS3HEN payload.

**Reply by pilgisayar63** [reply]
Thank you for your answer, do you think 4.90 is more stable or 4.91? I was using webman before, but I didn't use it for a long time because my system froze so much.

**Reply by aldostools** [reply]
Early betas of 4.91 were very unstable. Since BETA9 and BETA10 are very stable.

If your system is freezing check if you have this path `/dev_hdd0/tmp/system_plugin`. Delete this folder if you have it.

**Reply by pilgisayar63** [reply]
Thank you, I will definitely implement it.

**Comment by JuanzLzxt** [comment]
Amigo soy yo otra vez veras tengo un disco duro externo con juegos en formato iso lo que pasa esque cada ves q descargo un juego en internet lo meto a mi disco duro, luego conecto el disco duro a la consola le doy en actualizar en webman y se congela yo se que es por el beep podias decirme como desactivarlo? Porque en setup no veo ninguna opcion que diga beep la alarma que suena cuando le doy en actualizar porfas amigo :( o a que se debe el congelamiento tendra solucion? Antes no era asi :(

**Reply by aldostools** [reply]
Los beep se puden desactivar desde /setup.ps3 desmarcando la opción BEEP y guardando la configuración.
![image](https://github.com/user-attachments/assets/da37d220-9f45-4c7c-98ff-b920bcfdecdb)

**Comment by casiopea7** [comment]
hola soy nuevo acabo de instalar hen 4.9.1 y webMAN_MOD_1.47.47 pero cuando accedo a la carpeta de juegos se me queda congelada la consola y tengo que repetir el proceso hasta 3 y 4 veces alguna solucion?

**Reply by aldostools** [reply]
Solo al acceder a las carpetas de juegos de webMAN MOD, al montar el juego o cuando inicia luego de ejecutarlo?

Si es al acceder las carpetas, puede que el problema sea por alguna caratula muy grande o la cantidad de juegos en esa carpeta. En ese caso, es un problema de configuración del ambiente, ya que cuando navegas por el XMB aun webMAN no entra en función.

Si es al montar el juego, puede que sea por algun problema en el dispositivo de almacenamiento o el archivo del juego. Tambien puede ocurrir si se elimina el mapeo en memoria al idle_plugin.sprx. En este caso la consola se friza porque no encuentra el archivo idle_plugin usado como proxy para comunicar el XMB con webMAN MOD.

Si se friza al iniciar el juego o muestra pantalla negra, puede ser porque el juego no sea ISO o el juego esté guardado en un disco en NTFS o exFAT pero se haya dañado el archivo .ntfs que contiene la referencia a los sectores del archivo del juego en el disco. Otras razones pueden estar asociado al control dinámico de temperatura, consumo de memoria o inicio del juego mientras el FTP esta transfiriendo archivos.

**Reply by casiopea7** [reply]
nada mas entrar a la carpeta game se me queda congelado sin poder ni ejecutar el iso tengo que apagar la consola varias veces y al final consigue funcionar pero nuca a la primera  tengo 60 games puede ser por eso si en asi como lo soluciono?
